### DISCUSSION TITLE
1.1.35

### DISCUSSION THREAD

**Discussion by elihwyma** [wstyres/Zebra#2051] [Releases]
- Fix some issues regarding downloading packages due to Canister analytics. Installing a file from the file system will now work.
- Sadly we have removed Apollo support, obligatory fuck u/spez
- We made some images take up even less room on your device, now thats optimisation!

<hr /><em>This discussion was created from the release <a href='https://github.com/zbrateam/Zebra/releases/tag/v1.1.35'>1.1.35</a>.</em>

**Comment by burkybang** [comment]
Just curious, how was Apollo supported before this release? Was there an integration somehow?

(I'm deeply sad about Apollo being discontinued due to u/spez.)

**Reply by kirb** [reply]
The news carousel at the top of the Changes tab displayed recent posts about new and updated packages (it's currently disabled on our server as our way to protest). If Zebra detected Apollo as installed, it would open the link in Apollo, rather than the official Reddit app or the website. The downside is it was written to open *every* link in Apollo, even non-Reddit links, which Apollo just ignores. Since Apollo is shutting down anyway, we just removed support instead of fixing it, so in future we can reinstate the news carousel to point towards any website, not just Reddit. (Don't have a concrete idea of what that will look like just yet - still coming up with ideas.)

**Reply by burkybang** [reply]
Oh cool. Thanks for the explanation. I probably didn't notice because I use Evil Scheme to open Reddit links in Apollo anyway.

**Comment by daemonspudguy** [comment]
I love that "obligatory fuck u/spez". It's actually still probably too nice to that rancid little worm.

**Reply by elihwyma** [reply]
> I love that "obligatory fuck u/spez". It's actually still probably too nice to that rancid little worm.

I giggled to myself when I was writing those release notes. Such a shame how a good platform has been reduced to nothing.
